### DISCUSSION TITLE
Windows desktop  client, Does not save sound settings!

### DISCUSSION THREAD

**Discussion by 52rpm** [stoatchat/discussions#823] [General]
![rv](https://github.com/user-attachments/assets/2abc5fa5-f848-450b-b8a4-39b395bcbebb)

![rv1](https://github.com/user-attachments/assets/fb74e653-7777-4eeb-bb7b-d8588f9b81d5)

**Comment by ramazancet1nkaya** [comment]
Hi,
Same here, also tried on my macbook, same there too..
